### PR TITLE
MCR-1849 set increment before log output

### PR DESCRIPTION
**Pull request by bt1cu**
move increment before log output

### CODE REVIEW

File: mycore-solr/src/main/java/org/mycore/solr/index/statistic/MCRSolrIndexStatistic.java [hunk 1]
```diff
@@ -45,13 +45,15 @@ public MCRSolrIndexStatistic(String name) {
     }
 
     public long addTime(long time) {
+        accumulatedTime.addAndGet(time);
         LOGGER.debug("{}: adding {} ms", name, time);
-        return accumulatedTime.addAndGet(time);
+        return accumulatedTime.get();
```
Comment (yagee-de) [inline mycore-solr/src/main/java/org/mycore/solr/index/statistic/MCRSolrIndexStatistic.java]: should be addAndGet like before to be an atomic operation

File: mycore-solr/src/main/java/org/mycore/solr/index/statistic/MCRSolrIndexStatistic.java [hunk 2]
```diff
@@ -45,13 +45,15 @@ public MCRSolrIndexStatistic(String name) {
...
     }
 
     public int addDocument(int docs) {
+        documents.addAndGet(docs);
         LOGGER.debug("{}: adding {} documents", name, documents);
-        return documents.addAndGet(docs);
+        return documents.get();
```
Comment (yagee-de) [inline mycore-solr/src/main/java/org/mycore/solr/index/statistic/MCRSolrIndexStatistic.java]: should be addAndGet like before to be an atomic operation

**Review by yagee-de — COMMENTED** [review]
Will have to look into MCR-1849. Your changes should result in the same return value, as by definition of 'addAndGet()'

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on fa80a4a6265ae32598e9e90ee12e813540315492! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/3271-pull_request-584

**Review by kkrebs — APPROVED** [review]

**Pull request merged**